atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -416,8 +417,15 @@ private void loadWithTs(TableReference tableRef,
                             ThreadSafeResultVisitor v,
                             ConsistencyLevel consistency) throws Exception {
         List<Callable<Void>> tasks = Lists.newArrayList();
-        for (Map.Entry<InetSocketAddress, List<Cell>> hostAndCells : partitionByHost(cells,
-                                                                               Cells.getRowFunction()).entrySet()) {
+        Map<InetSocketAddress, List<Cell>> hostsAndCells =  partitionByHost(cells, Cells.getRowFunction());
+        int i = 1;
+        int size = hostsAndCells.keySet().size();
+        for (Map.Entry<InetSocketAddress, List<Cell>> hostAndCells : hostsAndCells.entrySet()) {
+            if (log.isTraceEnabled()) {
+                log.trace("Making request {} of {} for a loadWithTs call.  It is against host {} and table {}",
```

Comment:
This logging is currently a bit misleading in that we're not actually making the request yet. I created #712 as a PR against this branch with what I envisage.